contracts/lib/registries/IPAccountChecker.sol

```diff
@@ -23,7 +23,7 @@ library IPAccountChecker {
         uint256 chainId_,
         address tokenContract_,
         uint256 tokenId_
-    ) external view returns (bool) {
+    ) internal view returns (bool) {
```

Comment:
why external to internal?  any comments?

Reply:
I got this error from storage layout checker

```
contracts/lib/registries/IPAccountChecker.sol: Linking external libraries like `IPAccountChecker` is not yet supported
          Use libraries with internal functions only, or skip this check with the `unsafeAllowLinkedLibraries` flag 
          if you have manually checked that the libraries are upgrade safe
          https://zpl.in/upgrades/error-006
 ```
 
 The library is probably upgrade safe, but since the tests passed when I set it to internal and we are really using it as an internal library

Reply:
By that I mean it is imported into a contract and used as:

```
using IPAccountChecker for IIPAccountRegistry;
if (!IIPAccountRegistry($.ipAccountRegistry).isIpAccount(ipAccount)) {
       revert Errors.AccessController__IPAccountIsNotValid(ipAccount);
}
```